lighthouse-core/config/default-config.js

```diff
@@ -265,12 +265,13 @@ module.exports = {
       name: 'Performance',
       description: 'These encapsulate your web app\'s current performance and opportunities to improve it.',
       audits: [
-        {id: 'first-contentful-paint', weight: 5, group: 'perf-metric'},
-        {id: 'first-meaningful-paint', weight: 3, group: 'perf-metric'},
-        {id: 'first-cpu-idle', weight: 5, group: 'perf-metric'},
+        {id: 'first-contentful-paint', weight: 3, group: 'perf-metric'},
+        {id: 'first-meaningful-paint', weight: 1, group: 'perf-metric'},
+        {id: 'speed-index', weight: 4, group: 'perf-metric'},
         {id: 'interactive', weight: 5, group: 'perf-metric'},
-        {id: 'speed-index', weight: 1, group: 'perf-metric'},
-        {id: 'estimated-input-latency', weight: 1, group: 'perf-metric'},
+        {id: 'first-cpu-idle', weight: 2, group: 'perf-metric'},
```

Comment:
nbd but why did the order change?

Reply:
oh just aligning more with the order presented over there, I have no opinion I can revert